pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -73,6 +75,11 @@ type APISpec struct {
 	Denied []APIAccessRule `json:"denied,omitempty"`
 }
 
+// WasmSpec describes the security profile for all Dapr Wasm components.
+type WasmSpec struct {
+	StrictSandbox bool `json:"strictSandbox,omitempty"`
```

Comment:
Removed `omitempty` make  `StrictSandbox` as required. Mark the `WasmSpec` as optional. In this way, we can tell whether it is defined as false or undefined.

Reply:
Please keep it optional in both places.

1. Unless you use pointers, it is not possible to distinguish between an explicit false or undefined. I also don't think we need to know an explicit false vs undefined.
2. This would be requires if `wasm` is set, but if `wasm` is empty then it's not required. It doesn't seem right.

Reply:
Done.